src/spotify_webapi.c

```diff
@@ -34,8 +34,8 @@
 
 
 // Credentials for the web api
-static char *spotify_access_token;
-static char *spotify_refresh_token;
+static char *spotify_access_token = NULL;
```

Comment:
As far as I know this is not strictly required, globals are always zero-initialized. I doesn't hurt, of course, but to stay in line with the other code I suggest not having it.